x/did/module/module.go

```diff
@@ -168,23 +168,22 @@ func (AppModule) ConsensusVersion() uint64 { return 1 }
 
 // AppModuleSimulation functions
 
-// GenerateGenesisState creates a randomized GenState of the authz module.
+// GenerateGenesisState creates a randomized GenState of the did module.
 func (AppModule) GenerateGenesisState(simState *module.SimulationState) {
-	simulation.RandomizedGenState(simState)
 }
 
-// ProposalContents returns all the authz content functions used to
+// ProposalContents returns all the did content functions used to
 // simulate governance proposals.
 func (am AppModule) ProposalContents(simState module.SimulationState) []simtypes.WeightedProposalContent {
 	return nil
 }
 
-// RandomizedParams creates randomized authz param changes for the simulator.
+// RandomizedParams creates randomized did param changes for the simulator.
 func (AppModule) RandomizedParams(r *rand.Rand) []simtypes.ParamChange {
```

Comment:
anonymizing unused variables gives us bonus quality points !

```suggestion
func (AppModule) RandomizedParams(_ *rand.Rand) []simtypes.ParamChange {
```